cmfive.php

```diff
@@ -192,15 +292,15 @@ function installCoreLibraries()
     $out = 0;
     try {
         if (strtoupper(substr(PHP_OS, 0, 3)) === 'WIN') {
-            echo exec('mklink /J system composer\vendor\2pisoftware\cmfive-core\system',$msg,$out);
+            echo exec('mklink /J system composer\vendor\2pisoftware\cmfive-core\system', $msg, $out);
         } else {
-            echo exec('ln -s composer/vendor/2pisoftware/cmfive-core/system system',$msg,$out);
+            echo exec('ln -s composer/vendor/2pisoftware/cmfive-core/system system', $msg, $out);
         }
     } catch (Exception $e) {
         echo $e->getMessage();
         $out = 1;
     }
-    if ($out!==0) {
+    if ($out !== 0) {
         echo "\nFailed Linking for : \nsystem <---> composer/vendor/2pisoftware/cmfive-core\system";
```

Comment:
While we're making adjustments here please change the backslash at the end of this line to a forward slash

Reply:
OK